### PR TITLE
Documentation to include updating to rebar3 and updating with rebar3

### PR DESCRIPTION
**Pull request by PrajaktaPurohit**
Signed-off-by: Prajakta Purohit <prajakta@chef.io>

### Description

Add documentation for updating rebar3 and updating deps with rebar3

### Check List

- [x] All commits have been signed-off for the Developer Certificate of Origin. See <https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco>

### CODE REVIEW

File: doc/FrequentTasks.md [hunk 1]
```diff
@@ -81,3 +81,46 @@ copying files back from the dev-vm to the host.
...
+      and run sync to sync the changes with the dev-vm.
+    - Make sure to stop the sync after it is complete.
+  - In the dev-vm
+    - cd /host/src/oc_erchef #Repeat for every erlang app
+    - ./rebar3 update
+  - This will create an updated rebar.lock file.
+  - cp rebar.lock /dotfiles #dotfiles folder is shared on host
+  - On the host (mac)
+    - cp chef-server/dev/dotfiles/rebar.lock chef-server/src/oc_erchef
+- Repeat the above steps for all the erlang apps.
```
Comment (markan) [inline doc/FrequentTasks.md]: The indenting here is strange; probably should be nested under Updating.

File: doc/FrequentTasks.md [hunk 2]
```diff
@@ -81,3 +81,46 @@ copying files back from the dev-vm to the host.
...
+- Updating the erlang deps from the host machine
+  - cd chef-server/src/oc_erchef
+  - rm -fr \_build
+  - ./rebar3 update
+  - rm -fr \_build
+- The lockfile will be updated in place. But this can cause a
```
Comment (markan) [inline doc/FrequentTasks.md]: These also should be indented/nested with the 'updating'

File: doc/FrequentTasks.md [hunk 3]
```diff
@@ -81,3 +81,46 @@ copying files back from the dev-vm to the host.
...
+  lot of confusion if the \_build directory remains.
```
Comment (markan) [inline doc/FrequentTasks.md]: Worth adding the specific instruction to delete the _build directory

File: doc/FrequentTasks.md (hunk 2 above)
Comment (PrajaktaPurohit) [reply doc/FrequentTasks.md]: done

File: doc/FrequentTasks.md (hunk 1 above)
Comment (PrajaktaPurohit) [reply doc/FrequentTasks.md]: done

File: doc/FrequentTasks.md (hunk 3 above)
Comment (PrajaktaPurohit) [reply doc/FrequentTasks.md]: like in line 123?

**Pull request merged**